Added zip type

kilakewe opened the pull request:
Added zip folder type to types list

kilakewe committed:
Added zip type

Added zip folder type to types list

kirananto commented on src/data.ts:
```diff
@@ -3516,7 +3516,12 @@ export const data = [
+    {
+    "mime": "application/x-zip-compressed",
+    "name": "Compressed Archive (zip)",
     "fileType": ".google-apps.video"
```
Are you sure the filetype of zip is of `.google-apps.video` ?

kilakewe commented:
The bottom most is, I also corrected a typo in the Google apps video file type

Regards,

Gage Keenan

On 19/08/2021, at 7:43 AM, Kiran Anto ***@***.***> wrote:

﻿
@kirananto requested changes on this pull request.

In src/data.ts:

> @@ -3516,7 +3516,12 @@ export const data = [
   },
   {
     "mime": "application/vnd.google-apps.video",
-    "name": "Google VIdeo",
+    "name": "Google Video",
+    "fileType": ".google-apps.video"
+  },
+    {
+    "mime": "application/x-zip-compressed",
+    "name": "Compressed Archive (zip)",
     "fileType": ".google-apps.video"
Are you sure the filetype of zip is of .google-apps.video ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

kilakewe committed:
Update data.ts

Copied section for formatting, but forgot to add file extension to definition. Sorry for the confusion.

kilakewe reviewed (commented):
Added correct file type definition for Zip files.
